aries-site/src/layouts/main/Layout.js

```diff
@@ -127,6 +131,13 @@ export const Layout = ({
     { id: 'main', label: 'Main Content' },
   ].filter(link => link !== undefined);
 
+  const {wholeViewHistory, status, setStatus} = useContext(ViewContext) || undefined;
+
+  //every time a new page loads, initalize ready state to false, until app.js declares otherwise
+  useEffect(() => {
+    setStatus(false);
```

Comment:
setStatus feels a little vague. Can we rename this variable so that it is more clear that this relates to wether or not the page is ready to show update/new notifications